cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -42,10 +42,11 @@ var (
 		"sles-12-byol":        "suse/translate_sles_12_byol.wf.json",
 		"sles-15-byol":        "suse/translate_sles_15_byol.wf.json",
 		"rhel-6":              "enterprise_linux/translate_rhel_6_licensed.wf.json",
-		"rhel-6-byol":         "enterprise_linux/translate_rhel_6_byol.wf.json",
 		"rhel-7":              "enterprise_linux/translate_rhel_7_licensed.wf.json",
 		"rhel-8":              "enterprise_linux/translate_rhel_8_licensed.wf.json",
+		"rhel-6-byol":         "enterprise_linux/translate_rhel_6_byol.wf.json",
 		"rhel-7-byol":         "enterprise_linux/translate_rhel_7_byol.wf.json",
+		"rhel-8-byol":         "enterprise_linux/translate_rhel_8_byol.wf.json",
```

Comment:
@zmarano  -- Any thoughts on the name of the user-facing OS name? (`rhel-8-byol` vs `rhel-8-byos`)